src/openforms/js/components/admin/form_design/AuthPluginAutoLoginField.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+
+import Field from '../forms/Field';
+import {Checkbox} from '../forms/Inputs';
+
+const AuthPluginAutoLoginField = ({availablePlugins, selectedPlugins, selectedPlugin, onChange, errors}) => {
+    let filteredPlugins = availablePlugins.filter(plugin => selectedPlugins.includes(plugin.id))
+    const authCheckboxes = filteredPlugins.map(plugin => {
+        return (
+            <li key={plugin.id}>
+                <Checkbox
+                    name={`autoLoginAuthenticationBackend.${plugin.id}`}
```

Comment:
this form field `name` attribute must match with the respective field name in the API for change events to fire properly.

if `name="foo"`, then the event will be something like:

```js
event = {
    target: {
        name: 'foo',
        value: 'bar',
    },
}
```

That is what the entire state/reducer machinery relies on to update the local state and push changes to the backend.